File: src/main/java/Entrenador.java
public abstract class Entrenador extends SeleccionFutbol {

    protected int idFederacion;

    public Entrenador(int idFederacion, String nombre, String apellido, int edad) {
        super(idFederacion, nombre, apellido, edad);
        this.idFederacion = idFederacion;
    }

    //Instanciacion del singleton
    SeleccionFutbol seleccionFutbol = SeleccionFutbol.getInstanceSF();

    //Ya que tengo una clase abstracta y en la logica el entrenador planifica
    // el entrenamiento con metodos es de la clasePadre seleccionFutbol y al no poder instanciarlo
    //recurro al uso del singlentos.


    public void planificarEntrenamiento(boolean juegaPartido ,boolean jpe){

        juegaPartido= false;
        jpe=false;

        if(seleccionFutbol.jugar_partido && seleccionFutbol.jugar_partido_exterior){
            seleccionFutbol.MsgEntrenamiento(true);
        }else{
            System.out.print("No hay entrenamiento");
        }
    }

    @Override
    public void partidoFutbol() {

    }

    @Override
    public void entrenamiento() {

    }
}

File: SeleccionFutbol/src/Entrenador.py
from src.SeleccionFutbol import SeleccionFutbol

class Entrenador(SeleccionFutbol ) :
    def __init__(self, idFederacion, Nombre, Apellido, edad):
        super().__init__(idFederacion, Nombre, Apellido, edad)
        self.nombre=Nombre
        self.idFederacion= idFederacion

    def getNombre_entrenador(self):
            return self.nombre

    def get_Id_Entrenador(self):
       return self.idFederacion

    def set_idEntrenador(self, idFederacion):
        self.idFederacion1 = idFederacion

    def entrenar(self, partido, partido_exterior):
        if partido ==True & partido_exterior==True:
            print("Entrenador entrena equipo")
        else:
            print("No hay entrenamiento")
